osu.Game.Rulesets.Osu/Skinning/IHasApproachCircle.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+
+namespace osu.Game.Rulesets.Osu.Skinning
+{
+    /// <summary>
+    /// A common interface between skin component implementations which provide an approach circle.
+    /// </summary>
+    public interface IHasApproachCircle
```

Comment:
Is there a reason this isn't applied to `DrawableHitCircle`?

Reply:
Nope.